src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/CodeModel.java

```diff
@@ -122,4 +122,8 @@ public native final void insertRoxygenSkeleton() /*-{
       this.insertRoxygenSkeleton && this.insertRoxygenSkeleton();
    }-*/;
    
+   public native final int buildScopeTreeUpToRow(int row) /*-{
+      return this.$buildScopeTreeUpToRow && this.$buildScopeTreeUpToRow(row);
```

Comment:
When might we not have `this.$buildScopeTreeUpToRow` ? If we don't have it it looks like this will return `undefined` (so it won't throw right away but might later when we access `row_`).